packages/components/src/Collapsible/Collapsible/Collapsible.module.scss

```diff
@@ -39,6 +39,7 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
 .container {
```

Comment:
To investigate usages of the `clear` variant or lack of variant (not the same as `default`) and let Jess know.
Will create a follow up task for alignment pending the investigation (implementation OOS here).